src/index.ts

```diff
@@ -24,7 +24,7 @@ if (typeof PRERENDER === 'undefined') {
   ga('create', 'UA-128752250-1', 'auto');
   ga('set', 'transport', 'beacon');
   ga('set', 'dimension1', displayMode);
-  ga('send', 'pageview');
+  ga('send', 'pageview', '/index.html', { title: 'Squoosh' });
```

Comment:
Should this be `/` rather than `/index.html`?

Reply:
In analytics, we've been tracking index.html since the beginning of July. Either work, and I tried to keep it consistent to what we have right now.

In an ideal world, we'd trigger `/` for intro, and `/editor` for when the user is in the editor, but that's adding a bunch more we probably don't need until v2.